docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+
+services:
+  app:
+    build: 
+      context: .
+      dockerfile: Dockerfile
+      network: host
+    container_name: user-application
+    ports:
+      - "8080:8080"
+    environment:
+      - SPRING_PROFILES_ACTIVE=docker
+      - JAVA_OPTS=-Xmx512m -Xms256m
+      - SERVER_PORT=8080
+    restart: unless-stopped
```

Comment:
by default, bridge docker network must have access to outside networks (can download dependencies, for example)
that's weird that it required host, it should not imo